apps/pie-storybook/docblocks/component-status.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import '@justeattakeaway/pie-tag';
+import componentStatusesJson from '../component-statuses.json';
+// import { ComponentStatuses } from '../interfaces/component-statuses';
+import { tagVariantToStatusMap } from '../data/tag-variants-to-statuses-map';
+
+const componentStatuses = componentStatusesJson;
```

Comment:
Nit: Do you need to reassign this varible or can you just call the import `componentStatuses` directly?